Task1_dynamic_lists_str.cpp

```diff
@@ -0,0 +1,203 @@
+﻿// Task1_dynamic_lists_str.cpp : Этот файл содержит функцию "main". Здесь начинается и заканчивается выполнение программы.
+//
+
+#include "pch.h"
+#include <iostream>
+#include "LinkedList.h"
+using namespace std;
+
+#define PTR_SIZE sizeof(char *) 
+
+enum node {VALUE, NEXT_PTR, NODE_SIZE};
+
+void StringListInit(char*** list) {
+	*list = NULL;
+}
+void StringListAdd(char*** list, string str) {
```

Comment:
You are using char* everywhere. Does it make sense to use string for this function?